Source/WebCore/page/DragController.cpp

```diff
@@ -239,74 +238,90 @@ inline static bool dragIsHandledByDocument(DragHandlingMethod dragHandlingMethod
     return dragHandlingMethod != DragHandlingMethod::None && dragHandlingMethod != DragHandlingMethod::PageLoad;
 }
 
-bool DragController::performDragOperation(DragData&& dragData)
+DragDataInfo DragController::performDragOperation(DragData&& dragData, LocalFrame& frame)
 {
+    DragDataInfo dragDataInfo(false, std::nullopt, std::nullopt);
```

Comment:
Default initialized.
```suggestion
    DragDataInfo dragDataInfo;
```